types/atom/linter/config.d.ts

```diff
@@ -1,26 +0,0 @@
-import "../index";
```

Comment:
Why are you removing this?

Reply:
The initial thought was that we couldn't really assume that the developer is going to be building around Linter anymore. It doesn't really hold up to thinking it through further though. If they're already querying a Linter configuration variable, then they're already using Linter in whatever way.

Reply:
At the very least extending `ConfigValues` with well-known properties is "free" -- we don't lose anything by doing that. Hence there's no good reason to remove type information IMO.

There are, of course, some caveats. `config.get` will return `undefined` if `linter` is not present/is disabled.

Might extend those config values with `undefined` to reflect that, but that would create a lot of unnecessary noise wrt `undefined`. I would argue leaving the consumer to check for `atom.packages.isPackageActive('linter')` before accessing these settings is a saner choice, and we can't enforce that with types (can we?)